client/src/app/template-questionnaire/template-questionnaire-form/template-questionnaire-form.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component} from '@angular/core';
+import * as questionnaireSchema from './template-questionnaire-schema.json'
+import {MetadataFormConfig} from "../../metadata-schema-form/models/metadata-form-config";
+import {MetadataFormLayout} from "../../metadata-schema-form/models/metadata-form-layout";
+import {SpecimenGroupComponent} from "../specimen-group/specimen-group.component";
+import {DonorGroupComponent} from "../donor-group/donor-group.component";
+
+export const layout: MetadataFormLayout = {
+  tabs: [
+    {
+      title: "Spreadsheet Questionnaire",
+      key: "template-questionnaire",
+      items: [
+        "template-questionnaire.technology.ontologies",
+        {
+          keys: [
+            "template-questionnaire.identifyingOrganisms",
+            "template-questionnaire.donorsRelated",
+            "template-questionnaire.preNatalQuantity",
+          ],
+          component: DonorGroupComponent
+        },
+        {
+          keys: [
+            'template-questionnaire.specimenType',
+            'template-questionnaire.specimenPurchased'
+          ],
+          component: SpecimenGroupComponent
+        },
+        "template-questionnaire.experimentInfo",
+        "template-questionnaire.experimentTimecourse",
+        "template-questionnaire.protocols"
+      ]
+    }
+  ]
+};
+
+
+@Component({
+  selector: 'app-template-questionnaire',
+  templateUrl: './template-questionnaire-form.component.html',
+  styleUrls: ['./template-questionnaire-form.component.css']
+})
+export class TemplateQuestionnaireFormComponent {
+  templateQuestionnaireSchema: any = (questionnaireSchema as any).default;
+  questionnaireData: object = {
+    "identifyingOrganisms": ["Human"],
+    "specimenType": ["Primary Tissue"]
+  };
+  config: MetadataFormConfig = {
+    layout: layout,
+    submitButtonLabel: "Generate",
+    cancelButtonLabel: "Cancel"
+  }
+
+  onSave($event: object) {
+    console.log($event)
```

Comment:
@rolando-ebi the `$event` here is an object in the form
```
{
  "valid": true/false (against schema),
  "value": object (matching schema)
}
```
example:
```
{
  valid: true,
  value: {
    identifyingOrganisms: ["Human"],
    specimenType: ["Primary Tissue"],
    protocols: ["Enrichment", "Differentiation"],
    technology: {
      ontologies: [{
        text: "10x v3 sequencing",
        ontology: "EFO:0009898",
        ontology_label: "10x v3 sequencing"
      }]
    }
  }
}
```

The schema is builds against is [this](client/src/app/template-questionnaire/template-questionnaire-form/template-questionnaire-schema.json) file: